doc/modules/multiclass.rst

```diff
@@ -161,6 +161,10 @@ One-Vs-One
 
 :class:`OneVsOneClassifier` constructs one classifier per pair of classes.
 At prediction time, the class which received the most votes is selected.
+In the event of a tie (among two classes with an equal number of votes), the
+class with the highest sum_of_confidences (aggregated from pair-wise 
```

Comment:
I would rather say: "In the event of a tie (among two classes with an equal number of votes), it selects the class with the highest aggregate classification confidence by summing over the pair-wise classification confidence levels computed by the underlying binary classifiers."